yt_dlp/cookies.py

```diff
@@ -135,7 +155,13 @@ def _extract_firefox_cookies(profile, logger):
         cursor = None
         try:
             cursor = _open_database_copy(cookie_database_path, tmpdir)
-            cursor.execute('SELECT host, name, value, path, expiry, isSecure FROM moz_cookies')
+            if isinstance(container_id, int):
+                logger.debug(
+                    f'Only loading cookies from firefox container "{container}", ID {container_id}')
+                cursor.execute(
+                    f'SELECT host, name, value, path, expiry, isSecure FROM moz_cookies WHERE originAttributes="^userContextId={container_id}"')
+            else:
+                cursor.execute('SELECT host, name, value, path, expiry, isSecure FROM moz_cookies')
```

Comment:
In this case, load only cookies from "no container"

Reply:
done